{
  "debianbug": 641405,
  "description": "django.contrib.sessions in Django before 1.2.7 and 1.3.x before 1.3.1, when session data is stored in the cache, uses the root namespace for both session identifiers and application-data keys, which allows remote attackers to modify a session by triggering use of a key that is equal to that session's identifier.",
  "releases": {
    "buster": {
      "fixed_version": "1.3.1-1",
      "repositories": {
        "buster": "1:1.11.21-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "fixed_version": "1.3.1-1",
      "repositories": {
        "jessie": "1.7.11-1+deb8u3",
        "jessie-security": "1.7.11-1+deb8u5"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "1.3.1-1",
      "repositories": {
        "sid": "1:1.11.21-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "1.3.1-1",
      "repositories": {
        "stretch": "1:1.10.7-2+deb9u4",
        "stretch-security": "1:1.10.7-2+deb9u4"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}